### ISSUE TITLE
Parsing args incorrect when using "=" as a separator and the value contains ":"

### ISSUE THREAD

**Issue by vletoux** [gentilkiwi/mimikatz#123]
Example:
/value=D:123 is parsed as / value=D 123 and because "value" is needed and "value=D" is received, the parsing fails.

This is because : as a priority over =

cf https://github.com/gentilkiwi/mimikatz/blob/dc7661c7d0f1704f150f5cc6146db948eecc221d/modules/kull_m_string.c#L255-L256

Workaround: use ":" as an argument separator

**Comment by gentilkiwi** [comment]
You're right, I think I'll remove `=` separator... it was not really designed to be used.
